apps/menu-app/webapp/src/context/AuthContext.tsx

```
@@ -0,0 +1,229 @@
+        resolve({ accessToken: await getAccessToken() });
+      } else {
+        refreshAccessToken()
+          .then(async () => {
+            const accessToken = await getAccessToken();
+            resolve({ accessToken: accessToken });
+          })
+          .catch(() => {
+            appSignOut();
+          });
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)